_posts/2021-03-29-Encoding the Lending Library.md

```diff
@@ -0,0 +1,214 @@
+				</text>
+				<text color="blue">
+					<date-end>Oct 26 1949</date-end>
+					<date-start>Oct 26 1949</date-start>
+					<title>The Gambler</title>
+					<date-start>Oct 26</date-start>
+					<title>Walter Rathenau his life &amp; work by
+						
+						<author>Count Harry Kessler</author>
+						<publication-info>
+							<publisher>Gerald Howe Ltd</publisher>
+							<address>23 Soho Square London</address>
+							<year>1929</year>
+						</publication-info>
+					</title>
+				</text>
+				<text color="faded-blue">
+					<note>
+						<unclear>Also wants	
+							<author>Christina</author>
+						</unclear>
+						&amp;	
+						<author>Morand's</author>
+						Books
+					</note>
+				</text>
+			</borrowing-record>
+		</side>
+		<side type="reverse"/>
+	</div>
+</div>
+```
+
+### Organizing Information: Categories and Hierarchy
+In order to categorize the information, I created a root tag that nested the separate cards. Within a div element for each card, I separated the encoding further with a ```<heading>``` element and a ```<borrowing-record>``` element. This first element contains Wright's name/address, while the second contains the remaining information on the card, including specific borrowed books and notes. 
+
+The first question I encountered was how to encode the dates listed on the cards. While I considered creating an element for each book which would include the borrowing date, data on the book itself, and the return date, many of the books were listed under one borrow date, and it did not feel accurate to repeatedly include the same date if it was not written on the original document. I also considered nesting the information about a book inside the tags related to its borrow date, but chose not to as borrow/return dates were not consistently available for all entries. 
+
+This question led me to nest each section of text by color in an element ```<text>```, which has an associated attribute to note the color of the text, which implicitly carries information on when a given entry or set of entries was made. Within this element, I included ```<date-start>``` and ```<date-end>``` elements to differentiate between the borrow and return dates for each book. This allowed each date to exist as an individual element, rather than being explicitly connected to a particular title. I also included elements for the book's title, author, and publication information when provided. When text was circled, crossed out, or unclear, I added elements to reflect these aspects of the text.
+
+Finally, I tagged individuals mentioned within the entries and notes on the cards. In some cases, where an individual was referred to in relation to their own work, I chose to use the ```<author>``` tag. When the note referred to the behavior or actions of a given individual, I checked if they were also members of Shakespeare and Company and tagged them using ```<member-name>```, the same tag used in the heading to identify Wright.
```

Comment:
This is a great distinction, but again, you could unify both sets of tags by nesting within a `<person>` tag as I suggested above.